AvenueForTheArts/API/Endpoints/Endpoints.swift

```diff
@@ -15,7 +15,7 @@ extension API {
             var path: String {
                 switch self {
                 case .events:
-                    return "/events/"
+                    return "/events"
```

Comment:
keep the trailing slash, you will just waste time getting 302 redirected to it otherwise ;)